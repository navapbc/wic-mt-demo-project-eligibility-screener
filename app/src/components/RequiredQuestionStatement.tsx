import { Trans } from 'next-i18next'
import { ReactElement } from 'react'

export const RequiredQuestionStatement = (): ReactElement => {
  return (
    <p>
      <Trans i18nKey="asterisk" /> (
      <abbr className="usa-hint usa-hint--required">*</abbr>).
    </p>
  )
}

export default RequiredQuestionStatement
