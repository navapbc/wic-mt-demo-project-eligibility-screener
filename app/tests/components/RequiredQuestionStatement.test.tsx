import { render } from '@testing-library/react'

import RequiredQuestionStatement from '@components/RequiredQuestionStatement'

it('should match snapshot', () => {
  const { container } = render(<RequiredQuestionStatement />)
  expect(container).toMatchSnapshot()
})
