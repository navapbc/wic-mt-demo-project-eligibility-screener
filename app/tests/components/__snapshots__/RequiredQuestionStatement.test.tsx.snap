// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <p>
    All required questions are marked with an asterisk
     (
    <abbr
      class="usa-hint usa-hint--required"
    >
      *
    </abbr>
    ).
  </p>
</div>
`;
